src/admin/config.yml

```
@@ -431,6 +431,7 @@ collections:
     fields:
       - {label: "Layout", name: "layout", widget: "hidden", default: "layouts/project.njk"}
       - {label: "Project Name", name: "projectName", widget: "string", required: true, hint: "Suggestion: If this is a sub-page of a project, enter the name of the project here."}
+      - {label: "Permalink Override (Pattern: '/projects/my-project/')", name: "permalink", widget: "string", required: false, hint: "Hint: permalink must be unique."}
```

```suggestion
      - {label: "Override Default Project URL ", name: "permalink", widget: "string", required: false, hint: "Default URL is '/projects/[project-name]/'. To override enter a string including '/' at start and end (i.e. '/example-url/'). This must be unique."}
```

---

Thanks @jhung, how should we communicate that body field is required when permalink is entered?

---

@chosww I don't think it's necessary. I think if someone is modifying the permalink they would know there needs to be body content to be displayed at that URL.

One more adjustment to the  
```suggestion
      - {label: "Override Default Project URL ", name: "permalink", widget: "string", required: false, hint: "Default URL is '/projects/[project-name]/'. To override enter a string including '/' (slash) at start and end (i.e. '/example/url/'). This must be unique."}
```